.github/workflows/master_build_pr.yml

```diff
@@ -35,6 +42,70 @@ jobs:
             echo "$FILE"
             echo "Starting Docker"
             docker build -f $FILE -t $IMAGE_NAME .
+            imageScan $IMAGE_NAME
+            set +e
+          }
+          imageScan(){
```

Comment:
rename it to `dockerImageScan` or `containerImageScan`. With packer, `image` will be ambiguous